src/interfaces/IFundsRequestable.sol

```diff
@@ -157,19 +106,21 @@ interface IAllocatorConduit {
 
     /**
      *  @dev    Function to check if a withdrawal request can be cancelled.
+     *  @param  domain        The unique identifier of the domain.
      *  @param  asset         The asset to check.
      *  @param  fundRequestId The ID of the withdrawal request.
      *  @return isCancelable_ True if the withdrawal request can be cancelled, false otherwise.
      */
-    function isCancelable(address asset, uint256 fundRequestId) external view returns (bool isCancelable_);
+    function isCancelable(bytes32 domain, address asset, uint256 fundRequestId) external view returns (bool isCancelable_);
```

Comment:
@hexonaut Just fixed the example to work with the new interfaces, at least there, it is not clear how we should use the `domain` param. We could put a `require` to see if matches with the one in the structure or just ignore it as now, but not sure exactly the purpose of it here.